packages/zosjobs/src/GetJobs.ts

```diff
@@ -153,7 +153,7 @@ export class GetJobs {
         Logger.getAppLogger().trace("GetJobs.getJobsCommon()");
         ImperativeExpect.toNotBeNullOrUndefined(session, "Required session must be defined");
         let query = JobsConstants.QUERY_ID;
-        // params.execData = true; // always returning start and end time data
+        let includeExecData = params?.execData != false;
```

Comment:
Should we use `!==` here? (`params?.execData !== false`)

Another alternative to default to `true` if the value is not specified is to use `??`
`parms?.execData ?? true`